src/bp/ui-admin/src/Pages/Workspace/Bots.jsx

```diff
@@ -99,12 +99,24 @@ class Bots extends Component {
     )
   }
 
+  // TODO implement this properly
+  async requestPromotion(botId) {
+    const { data } = await api.getSecured({ timeout: 10000 })({
+      method: 'post',
+      url: `/admin/bots/${botId}/promote`
+    })
```

Comment:
I suggest using the standard api call, that one was specific to set the responseType and change the timeout :-P api.getSecured().post()

Reply:
yeah, well about that ... the `TODO implement this properlky` comment is pretty self explanatory